integration_tests/src/main/python/sort_test.py

```diff
@@ -107,8 +107,12 @@ def test_single_nested_orderby_with_limit(data_gen, order):
 def test_single_sort_in_part(data_gen, order):
     # This outputs the source data frame each time to debug intermittent test
     # failures as documented here: https://github.com/NVIDIA/spark-rapids/issues/2477
+    # Update: we are setting the number of slices with `numSlices` as the error appears to be
```

Comment:
So will there be another PR to update these comments and clean them up if this proves to be correct?

Reply:
Was trying to keep some context as to why we set `numSlices`, but yeah thought similar. The test fails sporadically so I'll forget to remove this comment. Maybe just point to the issue as a test comment?

Reply:
(which we already do, but tidy up)

Reply:
Ya, just say we are doing 12 slices because of ...

Reply:
ok edited!